ouroboros-network/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Breaking changes
 
+- Removes `updatePeerSharing` from `KnownPeers` module API
```

Comment:
We need to define what is public API, and what isn't (not export all the internals as we do right now). `KnownPeers` is an internal API, nobody needs to know about it, so this isn't really a breaking change.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/4646